assets/app/styles/_core.less

```diff
@@ -112,6 +120,9 @@ html, body {
       float: left;
       width: @navbar-project-menu-sm;
     }
+    .navbar-filter-widget {
+      white-space: nowrap;
```

Comment:
what's the smallest width this rule will apply at? will this cause problems if there are labels with long names or values displayed in the filter widget?